openapi.yaml

```
@@ -1279,6 +1279,18 @@ paths:
                 enable_object_detection:
                   type: boolean
                   description: Defines whether to enable [object detection](/chat/converse#digitization-and-object-detection) for the processed file. Object detection includes support for extracting tables and checkboxes.
+                enable_entity_detection:
+                  type: boolean
+                  description: If true, enables signature and barcode detection for the processed file.
+                  default: true
+                write_converted_image:
+                  type: boolean
+                  description: If true, the processed images for the document pages are saved in the storage system.
+                  default: true
+                write_thumbnail:
+                  type: boolean
+                  description: If true, thumbnails for the document's pages are saved in the storage system.
```

```suggestion
                  description: Defines whether to generate and save thumbnails of each image file generated when processing the document. One thumbnail--a small image representation of the larger image--per document page is saved.
```
Again assuming this setting aligns to these Reader profile settings https://platform.instabase.com/docs/automate-workflows/step-config-reference/process-files/index.html#general-processing

---

@FR4NKESTI3N Why would someone enable or not enable this?

---

Yup! The settings for `write_converted_image` and `write_thumbnail` aligns with reader profile defaults.